### PR TITLE
Bug 1809001: Remove fqdn from public ip creation

### PR DESCRIPTION
**Pull request by alexander-demichev**
Remove fqdn from public ip creation. `s.Scope.Network()` always returns nil and this caused panic.
 I will open a followup PR that removes unused code.

### CODE REVIEW

File: pkg/cloud/azure/services/publicips/publicips.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ func (s *Service) CreateOrUpdate(ctx context.Context, spec azure.Spec) error {
 				PublicIPAllocationMethod: network.Static,
 				DNSSettings: &network.PublicIPAddressDNSSettings{
 					DomainNameLabel: to.StringPtr(strings.ToLower(ipName)),
-					Fqdn:            to.StringPtr(s.Scope.Network().APIServerIP.DNSName),
```
Comment (enxebre) [inline pkg/cloud/azure/services/publicips/publicips.go]: can we just remove `Fqdn`?

Comment (alexander-demichev) [reply pkg/cloud/azure/services/publicips/publicips.go]: right :D my brains are boiling today

**Comment by enxebre** [comment]
/retest
/approve
@JoelSpeed PTAL

**Pull request merged**